### PR TITLE
fix(deps) Updating aiohttp. Fixing python 3.10 tests compatibility

### PR DESCRIPTION
**Pull request by leosmerling-hopeit**
Changelog:

- [x] Updated aiohttp version 
- [x] Removed aiojobs to spawn stream tasks. Replaced with asyncio.create_task
- [x] Update web integration tests for compatibility with latest pytest_aiohttp
- [x] Handle CancelledError on stream timeout

### CODE REVIEW

**Comment by leosmerling-hopeit** [comment]
@pcanto-hopeit I could find the opportunity to do an user test on the simple-example app.

I would ask if you can check all works (auth + streams) properly when reviewing this PR?

**Comment by pcanto-hopeit** [comment]
> @pcanto-hopeit I could find the opportunity to do an user test on the simple-example app.
> 
> I would ask if you can check all works (especially auth + streams) properly when reviewing this PR?

I make a user test in 3.9 python environment and all looks to work fine, authentication and streams, all the endpoints response as expected.

File: engine/build/y.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+y
```
Comment (pcanto-hopeit) [inline engine/build/y.txt:1]: Is this file necessary?

**Review by pcanto-hopeit — COMMENTED** [review]
Everything seems fine, but there is a file that I don't know if it corresponds to, I left a comment.

**Review by pcanto-hopeit — APPROVED** [review]
